### PR TITLE
usbhid: add support for mouse buttons

### PR DESCRIPTION
**Pull request by sago35**
Add mouse button support to USB HID.
It can be used as follows

```go
// LEFT + RIGHT
m.Click(mouse.Left | mouse.Right)

// LEFT and Move
m.Press(mouse.Left)
m.Move(100, 0)
m.Release(mouse.Left)
```

### CODE REVIEW

File: src/machine/usb/hid/mouse/mouse.go [hunk 1]
```diff
@@ -57,11 +66,34 @@ func (m *mouse) Move(vx, vy int) {
...
 	m.buf.Put([]byte{
-		0x01, 0x00, byte(vx), byte(vy), 0x00,
+		0x01, byte(m.button), byte(vx), byte(vy), 0x00,
+	})
+}
+
+// Cilck clicks the mouse button.
+func (m *mouse) Click(btn Button) {
+	m.Press(btn)
+	//time.Sleep(100 * time.Millisecond)
```
Comment (deadprogram) [inline src/machine/usb/hid/mouse/mouse.go]: ?

Comment (sago35) [reply src/machine/usb/hid/mouse/mouse.go]: Oops, I forgot to remove it.

**Comment by deadprogram** [comment]
@sago35 do you want to squash these commits, or shall I do it when I merge it?

**Review by deadprogram — APPROVED** [review]

**Comment by sago35** [comment]
@deadprogram Please squash merge.

**Pull request merged**